app/src/main/java/com/lucas/medicaltools/repository/MedicalToolsRepository.kt

```diff
@@ -5,9 +5,22 @@ import com.lucas.medical_equip.service.RetrofitBuilder
 import io.reactivex.Observable
 import retrofit2.Response
 
-class MedicalToolsRepository() {
+object MedicalToolsRepository {
+    var medicalToolCache: Observable<Response<List<MedicalTool>>>? = null
```

Comment:
Update this to not cache the observable, should cache the response, like you had it on your interactor.